.github/workflows/CI.yaml

```
@@ -322,14 +321,23 @@ jobs:
         with:
           file: ./lcov.info
 
-      - name: "Cache: Pack"
+      - name: "Cache: Pack to master"
         if: ${{ github.ref == 'refs/heads/master' }}
         continue-on-error: true
         run: |
           tar -cf /tmp/build_target.tar ./target
           tar -cf /tmp/build_cache.tar /tmp/cachepot
           tar -cf /tmp/build_cargo_registry.tar /tmp/cargo/bin /tmp/cargo/registry/cache /tmp/cargo/registry/index /tmp/cargo/git
-          mv /tmp/*.tar /root/cache/
+          mkdir -p /root/cache/
+
+      - name: "Cache: Pack to temp"
+        if: ${{ github.ref != 'refs/heads/master' }}
```

This condition might be more readable.
```suggestion
        if: github.event_name == 'pull_request'
```